types/fs-extra/OTHER_FILES.txt

```diff
@@ -1 +1 @@
-esm.d.mts
+lib/esm.d.mts
```

Comment:
I don't think this 'lib' is required, only proper mapping for exports (to be figured out)

Reply:
Well, actually, currently it's the only way to import this module without TS errors. For this to work the definitions have to have the same layout as the module. As soon as the other problem is fixed this may not be necessary any more but would it hurt to have the correct layout?

Reply:
Yeah, you're right, this doesn't change anything. Only the exposed paths will ever work.

Reply:
done